Gemfile

```diff
@@ -69,3 +69,5 @@ if (agent_path=ENV['AGENT_PATH'])
 else
    gem 'contrast-agent' if !!ENV['CI_TEST']
 end
+
+gem 'puma' # TODO: RUBY-1541; remove puma dependency for capybara
```

Comment:
Will this cause any bother with our conditional installations of puma for `puma_max` and `puma_min`?